lizard_auth_client/admin.py

```diff
@@ -4,6 +4,19 @@
 from lizard_auth_client import models
 
 
+class OrganisationAdmin(admin.ModelAdmin):
+    model = models.Organisation
+    list_display = ('name', 'unique_id', )
+    search_fields = ('name', )
+
+
+class UserOrganisationRoleAdmin(admin.ModelAdmin):
+    model = models.UserOrganisationRole
+    list_display = ('role', 'user', 'organisation', )
```

Comment:
Heb je iets van een testje gedaan of dit performt? Aandachtspunt is dat er allemaal foreign keys opgehaald kunnen worden.

In lizard-auth-server staat her en der iets met `select_related`, zoals hier: https://github.com/lizardsystem/lizard-auth-server/blob/master/lizard_auth_server/models.py#L645

Ook in de `admin.py` aldaar.
Hier dus even naar kijken.